quantum/leader.c

```diff
@@ -61,6 +63,9 @@ bool leader_sequence_add(uint16_t keycode) {
     leader_sequence[leader_sequence_size] = keycode;
     leader_sequence_size++;
 
+    if (leader_add_user()) {
```

Comment:
It might make sens to pass the keycode to the user function here.

Reply:
Thank you for the response. I have added the keycode as a parameter to the leader_add_user callback and updated the documentation accordingly.